gatsby-theme-portfolio-minimal/src/components/Project/index.tsx

```diff
@@ -87,6 +90,7 @@ export function Project(props: ProjectProps): React.ReactElement {
                 <GatsbyImage
                     className={classes.ProjectImageWrapper}
                     imgClassName={classes.ProjectImage}
+                    objectFit={props.data.image.objectFit || 'cover'}
```

Comment:
Can we remove the fallback `|| 'cover'`?

So you are able to specify no "object-fit" at all if you don't want to